lib/fluent/test/driver/filter.rb

```diff
@@ -37,6 +37,10 @@ def filtered_records
           @filtered.map {|_time, record| record }
         end
 
+        def filtered_time
+          @filtered.map {|_time, record| _time }
```

Comment:
```suggestion
          @filtered.map {|time, _record| time }
```

The prefix `_` means `unused` so that it should be added to `record` not `time` for this case.

Reply:
I've fixed it.